src/pages/guidelines/accessibility/overview.mdx

```diff
@@ -16,26 +16,51 @@ IBM firmly believes that web and software experiences should be accessible for e
 <AnchorLink>Carbon and accessibility</AnchorLink>
 <AnchorLink>Categories of disabilities</AnchorLink>
 <AnchorLink>Global accessibility standards</AnchorLink>
-<AnchorLink>Resources</AnchorLink>
 <AnchorLink>Tools</AnchorLink>
 
 </AnchorLinks>
 
+## Resources
+
+<Row className="resource-card-group">
+<Column colMd={4} colLg={4} noGutterSm>
+  <ResourceCard
+    subTitle="IBM Accessibility Checklist"
+    href="https://www.ibm.com/able/guidelines/ci162/accessibility_checklist.html"
+    >
+
+![IBM bee icon](images/bee.svg)
+
+  </ResourceCard>
+</Column>
+<Column colMd={4} colLg={4} noGutterSm>
+  <ResourceCard
+    subTitle="IBM accessibility quick guidance (Internal resource)"
+    href="https://w3.ibm.com/able/devtest/quick/"
+    >
+
+![IBM bee icon](images/bee.svg)
+
+  </ResourceCard>
+</Column>
+</Row>
+
 ## Carbon and accessibility
 
 Accessible design not only helps users with disabilities; it provides better user experiences for everyone.
-All Carbon components follow the [IBM Accessibility Checklist](https://www.ibm.com/able/guidelines/ci162/accessibility_checklist.html) which is based on WCAG AA, Section 508, and European standards. Carbon's patterns are perceivable, operable, and understandable to users, even when using a screen reader or other assistive technology.
 
-However, how you use Carbon also affects the accessibility of a product. Read this page to learn about accessibility in general. Then, while you're working, use the role-specific Carbon accessibility guidelines to help you make products that anyone can use.
+An accessible product should:
 
-For a product to be considered accessible, it should:
+- Give every user the same quality of experrience, regardless of ability
+- Adapt to users and situations
 
-- Give every user the same benefits, regardless of ability
-- Adapt to any user in any context
+Carbon components follow the [IBM Accessibility Checklist](https://www.ibm.com/able/guidelines/ci162/accessibility_checklist.html) which is based on WCAG AA, Section 508, and European standards. The Carbon team strives to write perceivable, operable, and understandable patterns for all users—including those employing a screen reader or other assistive technology.
+
+Individually accessible elements and components are just part of building accessible products. Use this guide to design and build products that anyone can use.
 
 ## Categories of disabilities
 
-Disabilities can be thought of as a mismatch between one's ability and one's environment, and can be broken into three main types:
+Disabilities can be thought of as a mismatch between one's ability and one's environment. There are three main types:
```

Comment:
Hmmm -- this is heady stuff. Wondering if we _need_ this though.

Reply:
I'd rather get rid of it, wanted to leave it in in case if was critical. If you're cool with getting rid of it I am too